st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -106,6 +109,28 @@ def match(self, action_alias_match_api, **kwargs):
             pecan.abort(http_client.BAD_REQUEST, str(e))
             return [self._match_tuple_to_dict(match) for match in e.matches]
 
+    @request_user_has_permission(permission_type=PermissionType.ACTION_ALIAS_HELP)
+    @jsexpose(arg_types=[str, str, int], body_cls=ActionAliasHelpAPI,
+    status_code=http_client.ACCEPTED)
```

Comment:
Also, `arg_types` in @jsexpose defines types for query parameters for the controller. It should not be used without query parameters defined in arguments list and I'm quite surprised the endpoint works at all.